syncTextures.js

```diff
@@ -216,14 +220,11 @@ async function syncTextures() {
   const localTexturePath = path.join(__dirname, 'textures', '/');  // use textureFolder variable
   const diff = lodash.difference(remoteTextures, localTextures);
 
-  
   if (!Array.isArray(diff)) {
     return Promise.reject('Error! diff is not an array!');
   } else if (Array.isArray(diff) && diff.length) {
     const agent = new https.Agent({ 
       keepAlive: true,
-      maxSockets: 1,
```

Comment:
because its actually watching for the file to be completed and closing the socket this just slows things down unnecessarily